src/lib/wicked/adapter.js

```diff
@@ -121,6 +134,41 @@ class WickedAdapter {
         });
     }
 
+    /**
+     * Update route files
+     *
+     * @param {Array} routes - routes to update
+     * @return {Promise<Connection,Error>} Promise that resolve to the added connection
+     */
+    async updateRoutes(routes) {
```

Comment:
For the following iteration, we might consider moving this logic to a separate class (`RoutesManager` or whatever).